src/stories/containers/Endgame/components/BudgetStructureSection/BudgetStructureSection.tsx

```diff
@@ -1,13 +1,68 @@
+import styled from '@emotion/styled';
+import { useThemeContext } from '@ses/core/context/ThemeContext';
+import lightTheme from '@ses/styles/theme/light';
 import React from 'react';
 import SectionHeader from '../SectionHeader/SectionHeader';
+import TotalBudgetContent from '../TotalBudgetContent/TotalBudgetContent';
+import type { WithIsLight } from '@ses/core/utils/typesHelpers';
 
-const BudgetStructureSection: React.FC = () => (
-  <div>
-    <SectionHeader
-      title="Endgame Budget Structure"
-      subtitle="Some simple but poignant text about what endgame budgets are about"
-    />
-  </div>
-);
+const BudgetStructureSection: React.FC = () => {
+  const { isLight } = useThemeContext();
+
+  return (
+    <Content>
+      <SectionHeader
+        title="Endgame Budget Structure"
+        subtitle="Some simple but poignant text about what endgame budgets are about"
+      />
+
+      <Card isLight={isLight}>
+        <TotalBudgetContent />
+        <div
+          style={{
```

Comment:
Can move this to stylecomponent

Reply:
Sorry, I didn't added a acclaration comment there, this is temporal just to cover the space of the chart that should be developed after this